sdk/search/azure-search-documents/swagger/Update-Codegeneration.ps1

```diff
@@ -0,0 +1,2 @@
+& (Join-Path $PSScriptRoot ".." ".." ".." ".." eng scripts Invoke-Codegeneration.ps1) -Directory $PSScriptRoot -AutorestOptions '--tag=package-2021-04-30-Preview-searchindex'
+& (Join-Path $PSScriptRoot ".." ".." ".." ".." eng scripts Invoke-Codegeneration.ps1) -Directory $PSScriptRoot -AutorestOptions '--tag=package-2021-04-30-Preview-searchservice'
```

Comment:
```suggestion
$gitRoot = git rev-parse --show-toplevel
& (Join-Path $PSScriptRoot $gitRoot eng scripts Invoke-Codegeneration.ps1) -Directory $PSScriptRoot -AutorestOptions '--tag=package-2021-04-30-Preview-searchindex'
& (Join-Path $PSScriptRoot $gitRoot eng scripts Invoke-Codegeneration.ps1) -Directory $PSScriptRoot -AutorestOptions '--tag=package-2021-04-30-Preview-searchservice'
```

Reply:
I'm going to save this change for a later date, mainly due to the need to know what `git rev-parse --show-toplevel` does compared to direct path hierarchy traversal (`..`)